apps/sms/test/marionette/conversation_test.js

```diff
@@ -16,10 +17,20 @@ marionette('Conversation Panel Tests', function() {
     '/mocks/mock_navigator_moz_mobile_message.js',
     '/mocks/mock_navigator_moz_contacts.js'
   ];
+  var apps = {};
+  apps[MessagesActivityCaller.ORIGIN] = __dirname + '/apps/activitycaller';
 
-  var client = marionette.client();
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'focusmanager.testmode': true
```

Comment:
I remember it was needed for every b2gdesktop test. Like we even [set it by default in the marionette python driver](https://dxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/geckoinstance.py#179). I don't know if that's still mandatory with Mulet.

Also, IIRC, this pref was useful when you switch the the keyboard to type. This is done in these tests, so my guess is that it's still needed.